docs/Executing-Tests.md

```diff
@@ -8,17 +8,19 @@ This repository contains a folder `shell` that contains a lot of scripts that he
 shell/run-a-single-test.sh
 ```
 
-## Required environment variabales
+## Environment variabales for the K6 Container
 
-The following env vars **MUST** be set in the K6 container. Pay attention to the descriptions of them, some of them are very important! The data from those variables is used to generate reports and compare products and versions of them. You must ensure that the data is accurate when providing the result report data to our analytics system!
+The following env vars **MUST** be present **in the K6 container**. Pay attention to the descriptions of them, some of them are very important! The data from those variables is used to generate reports and compare products and versions of them. You must ensure that the data is accurate when providing the result report data to our analytics system!
 
-* **GIT_REPO** - The repository we test, we use it to identify which product we test.
-* **GIT_BRANCH** - The branch of the repository.
-* **GIT_HASH** - The commit hash.
-* **GIT_TAG** - The tag if any was present.
-* **K6_TEST_RUN_ID** - This **MUST** be a **NEW** UUID version 4 for each test run!
-* **K6_TEST_RUNNER_HOSTNAME** - Required to identify the used environment.
-* **K6_TEST_ENVIRONMENT** - The name of the AWS environment used e.g. "Production 2.1"
+* **GIT_REPO** (required)- The repository we test, we use it to identify which product we test.
+* **GIT_BRANCH** (required) - The branch of the repository.
+* **GIT_HASH** (required) - The commit hash ID.
+* **GIT_TAG** (optional)- The tag if any was present, **SHOULD** be set. Without it you won't be able to filter your test data later by a convenient Git tag.
+
+## Environment Variables for your System
+
+* **SPRYKER_TEST_RUN_ID** (optional): This **MUST** be a **NEW** UUID version 4 for each test run or you will end up with duplicate identifiers and you won't be able to distinguish different test runs. If not provided a UUID will be generated when running the shell scripts.
+* **SPRYKER_TEST_ENVIRONMENT** - The name of the (AWS) environment used e.g. "Production 2.1". This should be an identifier that helps you to identify the environment that was used for the test.
 

```

Comment:
SPRYKER_TEST_RUNNER_HOSTNAME?

Reply:
SPRYKER_TEST_RUNNER_HOSTNAME is set automatically. I'm not sure if we really need to set it at all. What do you think?

Reply:
Would be nice to have it explained somewhere in Running a single test section since it will be required in this case

Reply:
Why? It is set in line 82. Running a single test executes also build_k6_docker_command() where it is set to the env vars of the container.